requirements/min-reqs.old

```diff
@@ -25,6 +25,7 @@ torchvision==0.14.0
 h5py==3.6.0
 laspy==2.0.0
 opencv-python==4.5.4.58
+pyarrow==15.0.0
```

Comment:
pyarrow<15.0.0 doesn't have an explicit dependency on numpy but fails if numpy>=2 is installed

Reply:
Hmmm does this raise the min required pandas version to 2.0 (e.g. in min-reqs.old)

Reply:
A good question. 2.0 is over 1 yr old, so it wouldn't be _horrible_ to make that the new min version. But we could also just throw pyarrow at any old fixed version in there for older pandas just to make sure it works. Or skip that specific test and just let pytest skip it.

Reply:
@isaaccorley what were the issues you had with pandas>=2?

Reply:
I'm guessing that was an issue with numpy 2, not pandas 2?

Reply:
Ah yep nevermind, he was mentioning "geopandas<1" and "numpy<2". I'm fine with adding pyarrow to min-reqs with a note that says to remove that when we upgrade pandas[parquet]>=2 as a min requirement.

Reply:
Done